templates/compose/evolution-api.yaml

```
@@ -11,11 +11,14 @@ services:
     image: 'evoapicloud/evolution-api:latest' # Change to specific version if needed.
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

```suggestion
    image: 'evoapicloud/evolution-api:v2.3.4'
```